Makefile.am

```diff
@@ -242,10 +242,16 @@ compliant:
 	git diff --quiet --exit-code src; ret=$$?; \
 	if [ $$ret -eq 1 ]; then \
 		echo "Code style issues found. Run 'git diff' to view issues."; \
+	fi; \
+	OUT=$$(git grep if\ \(.*\)$$ -- '*.c'); ret2=$$?; \
+	if [ $$ret2 -eq 0 ]; then \
+		echo "Missing brackets in single line if:"; \
+		echo "$$OUT"; \
 	elif [ $$ret -eq 0 ]; then \
```

Comment:
What if clang-format makes changes in the source files and there are no missing brackets in single if? Based on this logic both this messages would be shown at the same time, wouldn't they?
"Code style issues found. Run 'git diff' to view issues."
"No code style issues found."